picard/ui/mainwindow.py

```diff
@@ -799,8 +799,12 @@ def play_file(self):
 
     def open_folder(self):
         files = self.tagger.get_files_from_objects(self.selected_objects)
+        opened = set()
         for file in files:
-            QtGui.QDesktopServices.openUrl(self._openUrl(os.path.dirname(file.filename)))
+            url = self._openUrl(os.path.dirname(file.filename))
+            if not url in opened:
+                QtGui.QDesktopServices.openUrl(url)
+                opened.add(url)
```

Comment:
Upon consideration, I am wondering whether use of os.path.realpath is actually wrong (as opposed to unnecessarily).

I think @zas thinking was to avoid duplication of two directories which are mapped by symbolic links to the same directory - and at first sight this seems correct.

However I think this has an unintended consequence of displaying the real location rather than the one the user would be expecting. So for example:

* Suppose I have two artist directories mapped to the same directory - for the sake of argument suppose `/Music/Annie Lennox` and `/Music/Dave Stewart` are both symbolic links to `/Music/Eurythmics`.
* And suppose I have an Annie Lennox album loaded in Picard from  `/Music/Annie Lennox` and a Dave Stewart album loaded from `/Music/Dave Stewart`. If I select all the tracks and Open containing Folder I would expect to get two file explorer windows opening, one in each directory, but instead I get one file explorer window opening in `/Music/Eurythmics`.

So the results would not be as I would expect. Indeed it is possible that I have permissions to view the linked from directories but don't have security permissions to view the linked to directory - and I might not get a window opening at all.

So, on the whole I am thinking that it might be better to drop use of `os.path.realpath`.

@zas what do you think?